### PR TITLE
Tree-widget: fixes for itwinUI-react 3.13

### PR DESCRIPTION
**Pull request by MartynasStrazdas**
Fixes [#1021](https://github.com/iTwin/viewer-components-react/issues/1021)

### CODE REVIEW

File: packages/itwin/tree-widget/src/e2e-tests/CategoriesTree.test.ts-snapshots/Categories-tree-Density-enlarged-shows-outlines-when-focused-using-keyboard-2-chromium-linux.png
Comment (saskliutas) [inline packages/itwin/tree-widget/src/e2e-tests/CategoriesTree.test.ts-snapshots/Categories-tree-Density-enlarged-shows-outlines-when-focused-using-keyboard-2-chromium-linux.png:1]: Can you check why horizontal scroll position is different in updated screenshot?

File: packages/itwin/tree-widget/src/e2e-tests/CategoriesTree.test.ts-snapshots/Categories-tree-Density-enlarged-shows-outlines-when-focused-using-keyboard-1-chromium-linux.png
Comment (saskliutas) [inline packages/itwin/tree-widget/src/e2e-tests/CategoriesTree.test.ts-snapshots/Categories-tree-Density-enlarged-shows-outlines-when-focused-using-keyboard-1-chromium-linux.png:1]: Is it expected that scroll bar is not present in the new screenshot?

File: packages/itwin/tree-widget/src/e2e-tests/CategoriesTree.test.ts-snapshots/Categories-tree-Density-enlarged-shows-outlines-when-focused-using-keyboard-2-chromium-linux.png
Comment (MartynasStrazdas) [reply packages/itwin/tree-widget/src/e2e-tests/CategoriesTree.test.ts-snapshots/Categories-tree-Density-enlarged-shows-outlines-when-focused-using-keyboard-2-chromium-linux.png:1]: At all times the scroll is at the left most position.

**Review by saskliutas — APPROVED** [review]

**Comment by JonasDov** [comment]
There is a new iTwinUI version 3.14.0. I think it should be updated in this PR. https://github.com/iTwin/iTwinUI/releases/tag/%40itwin%2Fitwinui-react%403.14.0

**Pull request merged**